e2e/tests/components/marquee-wizard/marquee-wizard.e2e-spec.ts

```diff
@@ -85,6 +85,40 @@ describe('Marquee Wizard Tests', () => {
         expect(attr).toBeNull();
     });
 
+    it('should disable the next button when the step is invalid and disableNextWhenInvalid = true', async () => {
+        // enable validation
```

Comment:
I think this comment is misleading, you're setting the step to be invalid with the following line.